From 9492c30095f8aa60b9dde1c662adbd33568cd82f Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Thu, 18 Aug 2022 19:50:26 +0300
Subject: [PATCH] [SW-86869] gaudi3: rename disable pcie rr function

This patch renames disable pcie rr function name in rom recovery
to enhance code readability.

Change-Id: I2186e83420d689b35e3ec89a20c8fa460666f0a4
---

diff --git a/arch/gaudi3/arc/dev_security.h b/arch/gaudi3/arc/dev_security.h
index 8b29505..32d9893 100644
--- a/arch/gaudi3/arc/dev_security.h
+++ b/arch/gaudi3/arc/dev_security.h
@@ -69,6 +69,6 @@
 
 int secureboot_verify_image_hash384(uint8_t *exp_hash, unsigned char *image, uint32_t image_len);
 void security_rr_recovery_config(void);
-void security_rr_recovery_pcie_config(void);
+void security_rr_disable_pcie_config(void);
 
 #endif /* _GAUDI3_DEV_SECURITY_H_ */
diff --git a/arch/gaudi3/arc/recovery.c b/arch/gaudi3/arc/recovery.c
index c544974..60cd80e 100644
--- a/arch/gaudi3/arc/recovery.c
+++ b/arch/gaudi3/arc/recovery.c
@@ -133,7 +133,7 @@
 	if ((arcboot_rom_global.die_id && pcie_ctrl_val) ||
 		(!arcboot_rom_global.die_id && !pcie_ctrl_val)) {
 		disable_pif_isolation();
-		security_rr_recovery_pcie_config();
+		security_rr_disable_pcie_config();
 		disable_pcie_error_resp();
 		pcie_config();
 	}
diff --git a/arch/gaudi3/arc/security.c b/arch/gaudi3/arc/security.c
index f5d946e..a258183 100644
--- a/arch/gaudi3/arc/security.c
+++ b/arch/gaudi3/arc/security.c
@@ -737,7 +737,7 @@
 	security_rr_config_debug_if_hbw(&rr_ctx_hbw);
 }
 
-void security_rr_recovery_pcie_config(void)
+void security_rr_disable_pcie_config(void)
 {
 	security_rr_config_pcie_lbw(&rr_ctx_lbw);
 
